explore/src/main/scala/explore/notes/NotesTile.scala

```diff
@@ -6,96 +6,41 @@ package explore.tabs
 import cats.syntax.all.*
 import crystal.react.*
 import eu.timepit.refined.types.string.NonEmptyString
-import explore.Icons
 import explore.components.HelpIcon
+import explore.components.MarkdownEditor
 import explore.components.Tile
 import explore.components.ui.ExploreStyles
 import explore.model.ObsTabTileIds
-import explore.model.Observation
 import explore.model.enums.TileSizeState
 import japgolly.scalajs.react.*
 import japgolly.scalajs.react.vdom.html_<^.*
-import lucuma.core.util.NewType
 import lucuma.react.common.*
-import lucuma.react.markdown.ReactMarkdown
-import lucuma.react.markdown.RehypePlugin
-import lucuma.react.markdown.RemarkPlugin
-import lucuma.react.primereact.*
 import lucuma.refined.*
-import lucuma.ui.primereact.*
-import lucuma.ui.primereact.given
-import monocle.Focus
-import monocle.Lens
 
 object NotesTile:
 
-  def notesTile(obsId: Observation.Id, notes: View[Option[NonEmptyString]]) =
+  def notesTile(notes: View[Option[NonEmptyString]]): Tile[Unit] =
     Tile(
       ObsTabTileIds.NotesId.id,
       s"Note for Observer",
-      NotesTileState(notes.get.foldMap(_.value), Editing.NotEditing),
       bodyClass = ExploreStyles.NotesTile
-    )(NotesTileBody(obsId, notes, _), NotesTileTitle(obsId, notes, _, _))
-
-case class NotesTileState(notes: String, editing: Editing)
-
-object NotesTileState:
-  val notes: Lens[NotesTileState, String]    = Focus[NotesTileState](_.notes)
-  val editing: Lens[NotesTileState, Editing] = Focus[NotesTileState](_.editing)
+    )(_ => NotesTileBody(notes), (_, tileSize) => NotesTileTitle(notes, tileSize))
 
 case class NotesTileBody(
```

Comment:
Is it worth a component for this? could it be just inlined?

Reply:
I guess it could be inlined. But ObsTabTiles is a pretty busy file so I think this is cleaner.

Reply:
Oh, I guess you mean inlining `within` this file. Yeah, that would be simpler now that I look at the file. I'll do that.